docs/installation.md

```diff
@@ -79,6 +79,7 @@ The following arguments can be passed to the script:
 | `--skip-config`                  | `s`        | Do not create default configuration                                                                                                                                          | no                         |
 | `--skip-systemd` | `d`        | Preserves from Systemd service installation.                                                                                                                                 | no                         |
 | `--fips`                          | `f`        | Install the FIPS-compliant binary. See [this document](fips.md) for more details.                                                                                                                                                  | no                         |
+| `--install-hostmetrics`          | `H`        | Install the hostmetrics configuration to collect host metrics.                                                                                                               | no                         |
```

Comment:
It sounds strange for me. Wouldn't the following be better?

```suggestion
| `--install-hostmetrics`          | `H`        | Install the configuration to collect host metrics.                                                                                                               | no                         |
```

Reply:
Suppose it doesn't matter anymore :shrug:

Reply:
I took that from the install script - maybe we should say `hostmetrics receiver` there. Agree this looks a bit odd right now.